### ISSUE TITLE
aria-hidden and the remove "button"

### ISSUE THREAD

**Issue by plweil** [shentao/vue-multiselect#708]
I'm wondering what the reason is for assigning aria-hidden="true" to the remove button (actually an <i> element) for current selections:

              <span class="multiselect__tag">
                <span v-text="getOptionLabel(option)"></span>
                <i aria-hidden="true" tabindex="1" @keydown.enter.prevent="removeElement(option)"  
                @mousedown.prevent="removeElement(option)" class="multiselect__tag-icon"></i>
              </span>

When coded in this manner, the <i> element is completely removed from the accessibility tree, which would leave a screenreader user unable to delete a selection. I would also suggest using a button element instead of the <i>; this is exactly what buttons are for (and they're focusable by default). Using an svg icon with a title and maybe an aria-labelledby attribute would also assist the user in learning what the button does (right now there is nothing but a unicode "x" character in the css -- which is not a good place to supply information to the user. I know that you're working on v3.0; I'd be glad to offer suggestions/thoughts for accessibility improvements to maximize or at least improve the usability of MultiSelect for users with disabilities.

**Comment by shentao** [comment]
Accessibility is a big issue right now and is one of the main reasons the component is now being rewritten for the 3.0 release. Would you like to help me with making it more accessible? I’m already halfway through the rewrite. I will try to push the 3.0 branch this weekend.

**Comment by plweil** [comment]
I'd be glad to help!

**Comment by plweil** [comment]
Just checking in to see whether there is anything new on 3.0 yet. My initial suggestions will likely focus on the html; it might be good to incorporate/integrate these earlier rather than later.
